### PR TITLE
Irf uncertainty

### LINKED ISSUE #6: Using an external module to calculate uncertainty
Hey, @jhamman !

There's an old-ish looking repo that has a function to calculate uncertainty for the carbon decay curve that we'd like to use: https://github.com/gschivley/ghgforcing/blob/master/ghgforcing/ghgforcing.py

What do you think is the best way to use this code / create clear attribution? 

A couple of observations: 
- The repo doesn't look active, so it might be nice not to depend on it.  
- Right now, we only want to use the functions related to CO2, not CH4. Would be nice not to bring any mentions of CH4 into the ton-year repo. 

Fork? Copy and paste the functions of interest with attribution? Would love your take!

### PR DESCRIPTION
**Pull request by freyac**
This PR closes #6 by importing the [ghgforcing module](https://github.com/gschivley/ghgforcing/blob/master/ghgforcing/ghgforcing.py) functions to calculate uncertainty for the joos_2013 IRF curve.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -7,7 +7,7 @@ select = B,C,E,F,W,T4,B9
 
 [isort]
 known_first_party=carbonplan
-known_third_party=numpy,pytest,setuptools
+known_third_party=numpy,pandas,pytest,scipy,setuptools
```
Comment (jhamman) [inline setup.cfg:10]: If you want pandas/scipy to be added as dependencies to this project, also add them to the `requirements.txt.`

File: tonyear/ghgforcing.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+import numpy as np
+import pandas as pd
+from scipy.stats import multivariate_normal
+
+
+def CO2_AR5(t, **kwargs):
+    """Returns the IRF for CO2 using parameter values from IPCC AR5/Joos et al (2013)
+    Keyword arguments are parameter values.
+    """
```
Comment (jhamman) [inline tonyear/ghgforcing.py:34]: Add a more complete doc-string here. Something like:

```
    Parameters
    ----------
    t : int
        Length of the time horizon (years)
    Returns
    -------
    baseline_curve : np.ndarray
        Baseline curve in the form of an 1D array
```

**Review by jhamman — COMMENTED** [review]
License stuff looks spot on. Just a few comments on the code.

And let's add a few tests for these functions before merging.

**Comment by jhamman** [comment]
@freyac - when you return to this, #13 added some additional test infrastructure to this project, so you'll want to pull in those changes. 

```
git pull origin main
```
should do it!

**Comment by freyac** [comment]
Okay, @jhamman. I pulled from main, but since the CI tests are failing, I'm I thinking I messed something up along the way. 

I spent a few minutes poking around, but am thinking you might be able to quickly point me toward a solution!

**Pull request merged**